docker-compose.yml

```diff
@@ -0,0 +1,26 @@
+version: '3'
+services:
+  proxy:
+    build:
+      context: .
+      dockerfile: ./development/docker/Dockerfile
+    command: bash -c "npm install && npm run proxy"
+    volumes:
+      - .:/coco
+      - /coco/node_modules
+    environment:
+      DEV_CONTAINER: 1
+    ports:
+      - 7777:3000
+  dev-server:
+    build:
+      context: .
+      dockerfile: ./development/docker/Dockerfile
+    command: bash -c "npm install && npm run dev"
+    volumes:
+      - .:/coco
+      - /coco/node_modules
```

Comment:
For docker on windows, I had to use one more `VOLUME /usr/coco/bower_components` because of symbol links.

Reply:
Can we include this back in this PR? If it can't live along side these changes, we could make a `docker-compose-windows.yml` that had that change.